sammo/throttler_test.py

```diff
@@ -35,8 +34,9 @@ async def test_basic_call_limit(n_jobs, completion_time):
         jobs = [g.create_task(simple_job(i, throttler)) for i in range(n_jobs)]
     jobs = [j.result() for j in jobs]
     durations = [j["duration"] for j in jobs]
-    assert min(durations) <= completion_time
-    assert max(durations) <= completion_time
+    # provide a relaxed upper bound for max duration to account for differences
+    # in executors across test environments
+    assert max(durations) <= (completion_time * 2)
```

Comment:
@t-schn This relaxed upper bound seems to make all environments happy. MacOS seems to be the flakiest of the 3 runners.

Reply:
Thanks!